.github/workflows/release-python.yml

```diff
@@ -71,8 +82,16 @@ jobs:
           MACOSX_DEPLOYMENT_TARGET: "10.14"
         run: python -m cibuildwheel --output-dir wheelhouse
 
+      - name: Build MacOS Py39 Wheels
+        if: ${{ matrix.python == 'cp39' && matrix.buildplat[0] == 'macos-11' }}
+        env:
+          MACOS_TEST_SKIP: "*arm64"
+          CIBW_BUILD: cp39-macosx_*
+          MACOSX_DEPLOYMENT_TARGET: "10.14"
+        run: python -m cibuildwheel --output-dir wheelhouse
+
       - name: Build wheels
-        if: ${{ matrix.python != 'cp38' || matrix.buildplat[0] != 'macos-11' }}
```

Comment:
Should this be != cp39?

Reply:
That was the removed line, the new line covers both cp38 and cp39 since they run on macos11.